devtools/create_project/xcode.cpp

```diff
@@ -25,6 +25,7 @@
 
 #include <fstream>
 #include <algorithm>
+#include <sys/stat.h>
```

Comment:
I'm pretty sure MSVC doesn't feature these POSIX interfaces. Our create_project tool is aimed to be buildable under all platforms we support.

That said, there is probably a bigger issue: We usually generate project files for release sources beforehand. AFAICT this Xcode output depends on the location of static libraries during project file generation. Is there any way we can avoid this and have it pick up the locations when the user opens the project files?